### PR TITLE
Update ILA GUI docs

### PR DESCRIPTION
**Pull request by alonamid**
Based on the new AWS instructions: https://forums.aws.amazon.com/ann.jspa?annID=6352

The GUI looks nice, but it still break paramiko

### CODE REVIEW

**Comment by sagark** [comment]
Can you test it first?

File: docs/Advanced-Usage/Debugging/Debugging-Hardware-Using-ILA.rst [hunk 1]
```diff
@@ -60,27 +60,30 @@ Below is an example `PLATFORM_CONFIG` that can be used in the `build_recipes` co
...
+  cd nice-dcv-2019.0-7318-el7
+  sudo yum -y install nice-dcv-server-2019.0.7318-1.el7.x86_64.rpm
+  sudo yum -y install nice-xdcv-2019.0.224-1.el7.x86_64.rpm
+  sudo systemctl enable dcvserver
+  sudo systemctl start dcvserver
+  sudo passwd centos
+  sudo systemctl stop firewalld
+  dcv create-session --type virtual --user centos centos
+
+These commands will setup linux desktop pre-requisites, install the NICE DVC server, setup the password as ``centos``, disable firewalld,
```
Comment (a0u) [inline docs/Advanced-Usage/Debugging/Debugging-Hardware-Using-ILA.rst]: The `passwd` command is setting a password for the `centos` user, not the password to `centos`.  The user will be prompted for a passphrase.

Minor nit: Capitalize "Linux".

File: docs/Advanced-Usage/Debugging/Debugging-Hardware-Using-ILA.rst [hunk 2]
```diff
@@ -60,27 +60,30 @@ Below is an example `PLATFORM_CONFIG` that can be used in the `build_recipes` co
 Using the ILA at Runtime
 ------------------------
 
-Prerequisite: Make sure that ports 3121 and 10201 are enabled in the firesim AWS security group.
+Prerequisite: Make sure that ports 8443, 3121 and 10201 are enabled in the firesim AWS security group.
 
 In order to use the ILA, we must enable the GUI interface on our manager instance.
-This can be done by running the following commands:
+In the past, AWS had a custon ``setup_gui.sh`` script. However, this was recently deprecated due to compatibility
```
Comment (alonamid) [inline docs/Advanced-Usage/Debugging/Debugging-Hardware-Using-ILA.rst]:
```suggestion
In the past, AWS had a custom ``setup_gui.sh`` script. However, this was recently deprecated due to compatibility
```

File: docs/Advanced-Usage/Debugging/Debugging-Hardware-Using-ILA.rst (hunk 1 above)
Comment (alonamid) [inline docs/Advanced-Usage/Debugging/Debugging-Hardware-Using-ILA.rst]:
```suggestion
These commands will setup Linux desktop pre-requisites, install the NICE DVC server, ask you to setup the password to the ``centos`` user, disable firewalld,
```

**Pull request merged**